From 89aa167726d12f9ae0b53c01a2fdc54601fc3cc5 Mon Sep 17 00:00:00 2001
From: Jim Nelson <jim@yorba.org>
Date: Wed, 29 May 2013 18:52:09 +0000
Subject: Build with LibRaw 0.15: Closes #7012

LibRaw 0.15 removed document-mode processing, which, as their release
notes say, "no one uses".  Compile error were nothing more than
missing symbols -- Shotwell wasn't using it either.
---
(limited to 'src/photos/GRaw.vala')

diff --git a/src/photos/GRaw.vala b/src/photos/GRaw.vala
index f57886e..b7c3922 100644
--- a/src/photos/GRaw.vala
+++ b/src/photos/GRaw.vala
@@ -21,12 +21,6 @@ public enum Colorspace {
     XYZ = 5
 }
 
-public enum DocMode {
-    STANDARD = 0,
-    GRAYSCALE = 1,
-    GRAYSCALE_NO_WHITE_BALANCE = 2
-}
-
 public errordomain Exception {
     UNSPECIFIED,
     UNSUPPORTED_FILE,
@@ -167,10 +161,6 @@ public class Processor {
         throw_exception("adjust_sizes_info_only", proc.adjust_sizes_info_only());
     }
     
-    public void document_mode_processing() throws Exception {
-        throw_exception("document_mode_processing", proc.document_mode_processing());
-    }
-    
     public unowned LibRaw.ImageOther get_image_other() {
         return proc.get_image_other();
     }
@@ -244,7 +234,6 @@ public class Processor {
         // threshold
         output_params->half_size = half_size;
         // four_color_rgb
-        output_params->document_mode = GRaw.DocMode.STANDARD;
         output_params->highlight = GRaw.HighlightMode.CLIP;
         output_params->use_auto_wb = true;
         output_params->use_camera_wb = true;
--
cgit v0.9.2
